editoast/src/models/pathfinding.rs

```diff
@@ -42,6 +37,64 @@ pub struct Pathfinding {
     pub infra_id: i64,
 }
 
+#[derive(Clone, Debug, Serialize, Queryable, Insertable, Derivative, AsChangeset, Model)]
+#[derivative(Default(new = "true"))]
+#[model(table = "osrd_infra_pathmodel")]
+#[model(create, update)]
+#[diesel(table_name = osrd_infra_pathmodel)]
+pub struct PathfindingChangeset {
+    #[diesel(deserialize_as=i64)]
+    pub id: Option<i64>,
+    #[diesel(deserialize_as=uuid::Uuid)]
+    pub owner: Option<uuid::Uuid>,
+    #[diesel(deserialize_as=NaiveDateTime)]
+    pub created: Option<NaiveDateTime>,
+    #[diesel(deserialize_as=diesel_json::Json<PathfindingPayload>)]
+    pub payload: Option<diesel_json::Json<PathfindingPayload>>,
```

Comment:
I believe `diesel_json::Json` is only meant for table definitions.

`serde_json::Value` (aka `JsonValue`) is the one you should be using, it has the proper `ToSql`/`FromSql` and is *way* nicer to use